public/robots.txt

```diff
@@ -0,0 +1,6 @@
+User-agent: *
+Allow: /signup
+Allow: /rules
+Disallow: /stats
+Disallow: /lobby/*
+Disallow: /game/*
```

Comment:
I think you need a trailing return in this file.

Reply:
[Googles](https://developers.google.com/search/docs/crawling-indexing/robots/create-robots-txt) docs does not show a return. I will research this more and verify.

Thanks!

Reply:
@jerryh-jr it's not a specific thing for robots.txt but a general linting rule we have for the project that all files end with a trailing newline. That's also what the red circle symbol 🚫 in github is indicating; lack of newline ad space bad of file

Reply:
Yeah, my point was he might need one, or if google doesn’t want it to add an exception to the rule. Thanks @itsalaidbacklife 👍🏼